tests/test_main.py

```diff
@@ -1 +1,24 @@
-# write your code here
+from app.main import is_isogram
+
+def test_empty_string():
+    assert is_isogram("") is True
+
+
+def test_upper_case_letter():
+    assert is_isogram("Adam") is False
+
+
+def test_isogram_true():
+    assert is_isogram("playgrounds") is True
+
+
+def test_repeating_consecutive_letters():
+    assert is_isogram("look") is False
+
+
+def test_repeating_non_consecutive_letters():
+    assert is_isogram("loko") is False
```

Comment:
These tests work the same. You can add same letters in different cases.

Reply:
Thank you for the review. 
In this case, I tested non-consecutive https://prnt.sc/26p0w85, or do you think it doesn't make sense?
"same letters in different cases." I test in the "test_upper_case_letter" line #7 or do you mean something other?